src/libsyntax/ext/base.rs

```diff
@@ -696,7 +696,7 @@ pub struct ExpansionData {
     pub depth: usize,
     pub backtrace: ExpnId,
     pub module: Rc<ModuleData>,
-    pub in_block: bool,
+    pub no_noninline_mod: bool,
```

Comment:
I think this field could use a comment. The meaning of its name is not immediately obvious on its own, I'd say...